Packs/ApiModules/Scripts/NGINXApiModule/NGINXApiModule.py

```diff
@@ -208,18 +208,22 @@ def nginx_log_monitor_loop(nginx_process: subprocess.Popen):
         nginx_log_process(nginx_process)
 
 
+def test_nginx_web_server(port: int, params: Dict):
+    protocol = 'https' if params.get('key') else 'http'
```

Comment:
Let's wrap the code block in a try/except and then re-raise the exception with a message that states `Failed testing nginx server: {ex}`

Reply:
Done